packages/orchestrator-ui-components/src/components/WfoForms/UserInputForm.tsx

```diff
@@ -33,7 +33,8 @@ import {
     EuiHorizontalRule,
 } from '@elastic/eui';
 
-import { ConfirmDialogActions, ConfirmationDialogContext } from '@/contexts';
+import { ConfirmationDialogContext } from '@/contexts';
+import { ConfirmDialogHandler } from '@/contexts';
```

Comment:
Can be combined as a single line import.

Reply:
done